src/ThemeLoader.js

```diff
@@ -0,0 +1,98 @@
+import cssVarsPonyfill from 'css-vars-ponyfill';
+
+const buildURL = (cdn = '', themeId = '') => {
+    if (cdn === '' && themeId === '') return '';
+
+    if (cdn.charAt(cdn.length - 1) !== '/') {
+        return `${cdn}/${themeId}`;
+    }
+
+    return `${cdn}${themeId}`;
```

Comment:
shouldn't the url have .css at the end?

Reply:
I don't think so. Themes are identified by ID. So at the end we would end up in a URL like: `http://cdn.com/themes?id=12388123` , that returns a CSS . This is how I see

Reply:
So far the implementation of ThemeRoller service PoC assumes that we should request it like http://cdn.com/themes/THEME_ID.css
But it's something to discuss

Reply:
maybe it is good to add support for it when there is a cdn to support. Also it seems to be very 3rd party thing, and implementing it to this detail here is perhaps a bit too opinionated. What about having only one prop url, which can be constructed on the application level?